pages/c/_cluster/apps/charts.vue

```diff
@@ -155,6 +155,21 @@ export default {
 
       return out;
     },
+    flattenedRepoNames() {
+      const shownRepos = this.repoOptions.filter(repo => !this.hideRepos.includes(repo._key));
+      const reducedRepos = shownRepos.reduce((acc, c, i) => {
```

Comment:
That wont quite work here because `shownRepos` is an array of repo objects, defined as
```
this.$store.getters['catalog/repos'].map((r) => {
        return {
          _key:    r._key,
          label:   r.nameDisplay,
          color:   r.color,
          weight:  ( r.isRancher ? 1 : ( r.isPartner ? 2 : 3 ) ),
          enabled: !this.hideRepos.includes(r._key),
        };
   });
```

Reply:
I think I've just failed an interview question! Could always still do something like `shownRepos.map(s => s.label).join(', ')` but then it's a lines vs speed thing i guess. Could also include the `repoOptions.filter` in the `reduce`. Either way, it's fine as is.